app/src/main/java/org/systers/mentorship/models/HomeStatistics.kt

```diff
@@ -0,0 +1,36 @@
+package org.systers.mentorship.models
+
+import android.annotation.SuppressLint
+import android.os.Parcelable
+import com.google.gson.annotations.SerializedName
+import kotlinx.android.parcel.Parcelize
+
+/**
+ * This class represents statistics of the user actions on the app.
+ * @param name the name of the user
+ * @param pendingRequests number of pending requests
+ * @param acceptedRequests number of accepted requests
+ * @param completedRelations number of completed relations
+ * @param cancelledRelations number of cancelled relations
+ * @param rejectedRequests number of rejected requests
+ * @param achievements a list of upto 3 completed tasks
+ */
+
+@SuppressLint("ParcelCreator")
+@Parcelize
+data class HomeStatistics(
+        @SerializedName("name")
```

Comment:
While serialized name is not needed in this case, I still think its always a good idea to keep it around because in the future if the API changes and returns variables with a new name, we would just need to change its serialized name while the rest of our code can stay the same.

I'll remove it since you request it, though.